lib/editor-parser.ts

```diff
@@ -465,4 +479,9 @@ export class EditorParser {
             ],
         }));
     }
+
+    extractTextFromMarkUp(fragmentText) {
+        const match = fragmentText.match(/{{[^|]+\|([^}]+)}}/);
```

Comment:
suggestion: declare the regex as a global variable instead of declaring it here. The reason is because this regex will be compiled in every execution instead of being compiled only once.